dbt/adapters/trino/connections.py

```diff
@@ -373,6 +373,12 @@ def _escape_value(cls, value):
             raise ValueError("Cannot escape {}".format(type(value)))
 
 
+@dataclass
+class TrinoAdapterResponse(AdapterResponse):
+    query: str = ""
+    query_id: str = ""
```

Comment:
```suggestion
    query: str
    query_id: str
```
do we need empty string as default value here?

Reply:
I'm not totally sure. This is based on https://github.com/dbt-labs/dbt-snowflake/blob/5f4fbe0ea80798afe20c9ce4dfc19d3a3cbb94c6/dbt/adapters/snowflake/connections.py#L55